plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/form.tsx

```diff
@@ -72,11 +80,51 @@ const Form = < T extends AddressFormValues | ContactFormValues >( {
 		};
 	}, [ currentFields, currentFieldConfig, currentCountry, addressType ] );
 
+	const address2Field = addressFormFields.fields.find(
```

Comment:
I might be worth moving all of this field logic to its own file/component, it's overcrowding the form component.

Reply:
Thanks for pointing this out, @senadir. I moved the address 2 field related functionality into a separate component called `<Address2Field/>`, which I'm now using within the form.